public/javascripts/SVLabel/src/SVLabel/Tracker.js

```diff
@@ -154,6 +154,21 @@ function Tracker () {
         }
 
         var timestamp = new Date().getTime();
+        var data = {
```

Comment:
Adding a comment to remind you to delete this code when you don't need it anymore.

Reply:
Deleted it